core/shared/src/main/scala/org/http4s/headers/Sunset.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import cats.parse.Parser
+import org.typelevel.ci._
+
+object Sunset {
+  def parse(s: String): ParseResult[Sunset] =
+    ParseResult.fromParser(parser, "Invalid Sunset header")(s)
+
+  /* `Date = HTTP-date` */
+  private[http4s] val parser: Parser[`Sunset`] =
+    HttpDate.imfFixdate.map(apply)
```

Comment:
I fed you a ~~lie~~ terminological inexactitude: this one takes an HTTP date, with all the old cruft, rather than an imfFixDate.  The parser is already there, and we just need to flip the tests back to look more like `Date`'s.

Reply:
[Liar!](https://media2.giphy.com/media/12NlCFUvTokWXe/giphy.gif?cid=ecf05e47ggsdeb2xjjhewbaw1vgspjncvfhlzoqjxff90yus&rid=giphy.gif&ct=g)

Anyways, I believe this should match the Date header now